templates/default/apache2.conf.erb

```diff
@@ -11,19 +16,31 @@
 <% end %>
 <% end %>
   DocumentRoot    <%= node['nagios']['docroot'] %>
-  CustomLog       <%= node['apache']['log_dir'] %>/nagios_access.log combined
-  ErrorLog        <%= node['apache']['log_dir'] %>/nagios_error.log
+  CustomLog       <%= node['nagios']['log_dir'] %>/apache_access.log combined
+  ErrorLog        <%= node['nagios']['log_dir'] %>/apache_error.log
 
 <% if node.platform_family == 'debian' && node['nagios']['server']['install_method'] == 'package'-%>
-  Alias /stylesheets /etc/<%= node['nagios']['server']['vname'] %>/stylesheets
+  Alias /<%= node['nagios']['server']['vname'] %>/stylesheets /etc/<%= node['nagios']['server']['vname'] %>/stylesheets
 <% end -%>
-  ScriptAlias <%= node['nagios']['cgi-path'] %> <%= node['nagios']['cgi-bin'] %>
-  Alias /<%= node['nagios']['server']['vname'] %> <%= node['nagios']['docroot'] %>
+  ScriptAlias /<%= node['nagios']['server']['service_name'] %><%= node['nagios']['cgi-path'] %> <%= node['nagios']['cgi-bin'] %>
 
   <Directory "<%= node['nagios']['cgi-bin'] %>">
-     Options ExecCGI
+    Options ExecCGI
+    AllowOverride None
+    Order allow,deny
+    Allow from all
```

Comment:
This won't work with Apache 2.4 in RHEL 7 / Ubuntu 14.04:

http://httpd.apache.org/docs/2.4/upgrading.html